frontend/src/libs/components/table/libs/helpers/get-cell-click-handler.helper.ts

```diff
@@ -0,0 +1,30 @@
+import { checkIsActionCell } from './check-is-action-cell.helper.js';
+
+type Parameters = {
```

Comment:
It's built-in type [Parameters](https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype) 🥲